valhalla/baldr/edge_elevation.h

```diff
@@ -0,0 +1,84 @@
+#ifndef VALHALLA_BALDR_EDGEELEVATION_H_
+#define VALHALLA_BALDR_EDGEELEVATION_H_
+
+namespace valhalla {
+namespace baldr {
+
+// Use elevation bins of 8 meters to store mean elevation. Clamp to a range
+// from -500 meters to 7683 meters.
+constexpr uint32_t kMaxStoredElevation = 1023;
+constexpr float kElevationBinSize = 8.0f;
```

Comment:
@dnesbitt61 I see your point - I was thinking more of a short, relatively flat segments. but 12 bits sounds good! Let's do that.